decentraland/explorer-desktop
# refactor: updated display setttings to latest unity-renderer version

## What does this PR change?

Its a refactor of desktop `DisplaySettings` and `SettingsDesktop` to adapt to the latest settings persistence architecture from unity-renderer.

## How to test the changes?

1. Open the latest desktop build
2. Change your settings (general, quality & audio) to anything you'd like, preferably something different than default settings
3. Close the settings menu
4. Open desktop client with these changes (refactor/settings-persistence)
5. Open the settings menu, you should have the previous settings you changed (general, quality & audio)
6. Change any setting you like
7. Close the settings menu
8. Restart the app
9. You should have the previous settings
10. Go to quality settings then pick any preset from the dropdown (low, medium, high, ultra)
11. Restart the app
12. The preset dropdown should hold whatever preset you chose

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
